errors.toml

```diff
@@ -0,0 +1,405 @@
+AUTOGENERATED BY github.com/pingcap/tiup/components/errdoc/errdoc-gen
```

Comment:
The errors.toml needs to be generated again due to https://github.com/pingcap/tiup/pull/851/commits/f35a0dd68405652bd4e1a299ae7842ca34703f8e.